oak_crypto/src/hpke/mod.rs

```diff
@@ -40,15 +41,19 @@ pub(crate) type EncappedKey = <Kem as KemTrait>::EncappedKey;
 /// <https://www.rfc-editor.org/rfc/rfc9180.html#name-encryption-and-decryption>
 const MAX_SEQUENCE_NUMBER: u128 = (1 << (8 * AEAD_NONCE_SIZE_BYTES)) - 1;
 
-pub(crate) struct KeyPair {
+pub fn gen_kem_keypair() -> (PrivateKey, PublicKey) {
```

Comment:
you're not using this fn anywhere else, why make it public?

Reply:
I am in fact using it in the restricted kernel crate